olive/platform_sdk/qualcomm/env.py

```diff
@@ -0,0 +1,98 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import os
+import platform
+from pathlib import Path
+
+from olive.platform_sdk.qualcomm.constants import SDKTargetDevice
+
+
+class SDKEnv:
+    def __init__(self, sdk: str, root_env_name: str, target_arch: str = None, dev: bool = False):
+        self.sdk = sdk
+
+        self.root_env_name = root_env_name
```

Comment:
>root_env_name [](http://example.com/codeflow?start=29&length=13)

two questions:
1. need we check whether the root_env_name has value?
2. need we check whether the sdk_root_path exist or not?

Reply:
Updated in latest commit.